packages/fxa-payments-server/src/routes/Product/index.test.tsx

```diff
@@ -346,16 +346,13 @@ describe('routes/Product', () => {
   it('displays an error if payment submission somehow silently fails', async () => {
     const createToken = jest.fn().mockResolvedValue({});
     const {
-      container,
       getByTestId,
       findByTestId,
       queryByTestId,
-      debug,
     } = await commonSubmitSetup(createToken);
     fireEvent.click(getByTestId('submit'));
     await findByTestId('error-payment-submission');
-    fireEvent.click(getByTestId('dialog-dismiss'));
-    expect(queryByTestId('error-payment-submission')).not.toBeInTheDocument();
+    expect(queryByTestId('error-payment-submission')).toBeInTheDocument();
```

Comment:
I think this `expect()` ends up redundant, since the preceeding `await findByTestId()` will fail the test after a little while if the element isn't found.